translations/fritzing_ru.ts

```diff
@@ -2186,17 +2186,17 @@ It is also possible to choose a connector as a ground fill seed by right-clickin
     <message>
         <location filename="../src/mainwindow/mainwindow.cpp" line="2003"/>
         <source>%1 - [%2]</source>
-        <translation>%1 - [%2]</translation>
+        <translation>%1 — [%2]</translation>
     </message>
     <message>
         <location filename="../src/mainwindow/mainwindow.cpp" line="1433"/>
         <source>No Sketch found in &apos;%1&apos;</source>
-        <translation>Не найдено эскизов в &apos;%1&apos;</translation>
+        <translation>Не найдено эскизов в «%1»</translation>
```

Comment:
Please conserve special codes like `&apos;  `

Reply:
OK, though according to the Russian language rules these quotes should be used: «»